### ISSUE TITLE
One typo and one issue

### ISSUE THREAD

**Issue by vadamovsky** [Mediawiki-wysiwyg/WYSIWYG-CKeditor#35]
File:
CKeditorLinker.php

Line 151:
                      } elseif( found ) {

Should be:
                      } elseif( $found ) {

And one issue:

You can add images by specifying only width (we have used this style on our wiki):
[[image:picture.png|none|120px]]
The height is calculated automatically to maintain the same width/height ratio.

In the latest version of CKeditor the height is replaced with the real image height:
[[File:picture.png|none|120x600px|picture.png]]

You should not add the real image height. Instead, you should calculate:
newHeight = realHeight \* usedWidth / realWidth

**Comment by riparap** [comment]
It seems that in MW size of image can be defined in many different ways. I have removed use of real size of image, because it conflicts the way MW handles size of image. 

I have also modified image dialog further (version: 1.5.6_0 [B551+31.12.2014]):
canceled use of default size with images; width and height can both be specified, only one of them or none of them; size definition "upright"; horizontal alignment "none"; vertical alignments "baseline, sub, super, top, text-top, middle, bottom, text-bottom"; link target (external or internal); disable link.
